draft-ietf-rtcweb-jsep.xml

```diff
@@ -2501,6 +2551,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <t>The &lt;proto&gt; field MUST be set to exactly match the
             &lt;proto&gt; field for the corresponding m= line in the
             offer.</t>
+
+            <t>If codec preferences have been set for the
+            associated RtpTransceiver, media formats MUST
+            be generated in the corresponding order.</t>
```

Comment:
I'm not reading this in context but this should only be for Offer not answer. Perhaps it is OK as it.